homeassistant/components/smappee/strings.json

```diff
@@ -8,7 +8,7 @@
         "abort": {
             "authorize_url_timeout": "Timeout generating authorize url.",
             "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
-            "missing_configuration": "The component is not configured. Please follow the documentation."
+            "missing_configuration": "The component is not configured. Please follow the documentation, see [Smappee documentation](https://www.home-assistant.io/integrations/smappee)."
```

Comment:
We cannot put urls in translation strings. Please revert this change.

Reply:
Commit got reverted. I would like to use the description_placeholders argument from the async_abort method (missing configuration reason) but can't seem to get there by using the AbstractOAuth2FlowHandler class.

Reply:
I think that we should capture this in a more generic way in the OAuth2 flow helper. Ideally integrations only end up using references when using OAuth2.

Reply:
So for now I'd better not use placeholders in the strings/translations?